packages/storybook8/stories/Composites/ChatComposite/Docs.mdx

```diff
@@ -79,8 +78,8 @@ we support hiding the `Topic Header`:
 ## Participant Pane
 
 <SingleLineBetaBanner />
-You can also show a `Participant Pane` to the right hand side of the chat that has support for removing users. The Participant
-pane is a `beta` feature. Currently it does not support mobile views and is subject to breaking changes.
+You can also show a `Participant Pane` to the right hand side of the chat that has support for removing users. The
```

Comment:
Should 'The' be on the line below?

Reply:
I'm not sure why I have these changes, it looks like linter has applied some styling changes

Reply:
The text in Storybook looks okay though
![image](https://github.com/user-attachments/assets/fd23c472-b66d-4b7b-994e-7a97e66cfd74)